src/app.py

```diff
@@ -56,7 +56,9 @@ def post_zone(data):
 @app.route('/v1/zones/<id>', methods=['DELETE'])
 @api_response()
 def delete_zone(id):
-    Zone.objects.get(pk=id).delete()
+    zone = Zone.objects.get(pk=id)
+    zone.enabled = False
```

Comment:
maybe add a field - deleted ?
i thought about enabled to be able to enable/disable the zone temporarly
deleted means delete it forever